### DISCUSSION TITLE
Feature request: Select customers from contacts group

### DISCUSSION THREAD

**Discussion by TheRealRWJ** [baimard/gestion#325] [Ideas]
# Thanks

Thank for your effort for this great app.
Know it is very appreciated
And thanks for your quick answer on my previous reported issue
# Another Feature request:
Nextcloud makes it possible to organise your contacts in contact groups.
Can you please make is possible to select customers from one or more of these contacts groups

Once again Thanks in advance

**Comment by baimard** [comment]
Dear, it's in roadmap : https://github.com/baimard/gestion/discussions/127
